### DISCUSSION TITLE
Topmoney

### DISCUSSION THREAD

**Discussion by SomniumWed** [Crownicles/Crownicles#1382] [Suggestions fermées]
Rajouter un topmoney, avec pourquoi pas "joueur le plus riche" (on enleve pourfi du top bien évidement)

**Comment by Chevalier2** [comment]
Tu proposes un topmoney avec "joueur le plus riche" mais tu dis d'enlever le joueur le plus riche de ce top ?

**Comment by Stitch-seagull** [comment]
Personnellement je trouve que ce n'est pas l'objectif, ça pousse les gens à économiser des sommes monumentales ce qui est complètement inutile, le but c'est de récolter le plus de points. En plus il y a déjà des mini-events sur le sujet

**Comment by Nargacaura** [comment]
Perso, je n'vois pas vraiment l'intérêt de rajouter cette feature... Un top au niveau des points suffit largement (après, un top au niveau des levels peut être plus intéressant si on veut comparer ceux qui ont des plus hauts levels par rapport au joueur, sachant qu'un top des guildes peut éventuellement arriver dans une màj ultérieure).

**Comment by BastLast** [comment]
Hello je viens pour dire ma part et je rejoins à peu près les avis les deux derniers Avicii dessus pour moi il ne faut pas rajouter ce genre de classement annexe tout simplement parce que le budget de gagner des points et que tout le jeu est conçu autour de ce but là et que on doit plutôt rajouter des fonctionnalités qui permettent de rendre ce but-là fun pour tout le monde plutôt que d'encourager des des quêtes comme ça annexe parce que ça change complètement la manière dont certaines personnes jouent et que ça impacte les gens qui veulent profiter du jeu normalement dans le sens où il y a des joueurs très haut niveau par exemple qui se retrouve très bas dans l'emplacement parce qu'il joue uniquement pour l'argent enfin bref cueillez quelques énergumène qui s'amusent à le faire ça pose aucun problème mais j'ai pas du tout envie d'encourager ça à grande échelle

**Reply by SomniumWed** [reply]
Oui je comprends totalement 👌
